docs/_docs/other-guides/cueweb.md

```
@@ -25,158 +25,428 @@ This guide provides an introduction to the CueWeb system, the web-based version
+
+* **Advanced monitoring options:** Un-monitor jobs selectively or in bulk, providing flexibility in data visualization (see Figures 16 and 17).
+
+**Figure 16: Un-monitoring selected jobs (data table before un-monitor selection)**
+![Un-monitoring selected jobs (before)](/assets/images/cueweb/cueweb_cuetopia_monitor_jobs_un-monitoring_selected_jobs-before.png)
+
+**Figure 17:  Un-monitoring selected jobs (data table after un-monitor selection)**
+![Un-monitoring selected jobs (after)](/assets/images/cueweb/cueweb_cuetopia_monitor_jobs_un-monitoring_selected_jobs-after.png)
+
+* **Detailed inspections:** A pop-up detail view for layers and frames associated with a selected job (see Figure 18), offering deep dives into specific frame logs as shown in Figure 19.
+
+**Figure 18: Pop-up window to view layers and frames information **
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)